setup.py

```diff
@@ -2,7 +2,7 @@
 from setuptools import setup, find_packages
 import versioneer
 
-INSTALL_REQUIRES = ["xarray", "netcdf4", "typhon", "tqdm", "statsmodels"]
+INSTALL_REQUIRES = ["xarray", "netcdf4", "typhon", "tqdm"]
```

Comment:
I think we need `statsmodels` too, it's just that we don't have a test for @annalea-albright's linear_RH mixed-layer height yet, so this wouldn't get picked up. But I'll help fix this when we talk about git stuff in a moment